src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleFormUtility.ts

```diff
@@ -36,9 +36,9 @@ export function getConditionSecondaryText(
     values: unknown[]
 ): string {
     if (type === 'NumericRange') {
-        return `${values[0]} ${i18n.t('min')} ${i18n.t('to')} ${
+        return `${values[0]} (${i18n.t('min').toLowerCase()}) ${i18n.t('to')} ${
             values[1]
-        } ${i18n.t('max')}`;
+        } (${i18n.t('max').toLowerCase()})`;
```

Comment:
I just had a thought, does toLowerCase do the right thing across all languages? I guess it must, but I hadn't ever thought about that.

Reply:
Good question, I don't think JS knows which language you are inputting. I can take a look into it, but my guess is if there are no lower/upper case letters in a specific language the string won't change.

Reply:
Looks like it is not safe (https://stackoverflow.com/questions/30762886/is-javascripts-touppercase-language-safe#:~:text=JavaScript%27s%20toUpperCase%20%28%29%20method%20is%20supposed%20to%20use,is%20supposed%20to%20be%20I%20in%20all%20cases.). I'll add locales for this.